trace/export.go

```diff
@@ -56,16 +56,22 @@ func UnregisterExporter(e Exporter) {
 
 // SpanData contains all the information collected by a Span.
 type SpanData struct {
-	SpanContext
+	SpanContext  SpanContext
 	ParentSpanID SpanID
 	Name         string
 	StartTime    time.Time
-	EndTime      time.Time
+	Duration     time.Duration
 	// The values of Attributes each have type string, bool, or int64.
-	Attributes    map[string]interface{}
-	Annotations   []Annotation
-	MessageEvents []MessageEvent
-	Status
+	Attributes      map[string]interface{}
+	Annotations     []Annotation
+	MessageEvents   []MessageEvent
+	Status          Status
 	Links           []Link
 	HasRemoteParent bool
 }
+
+// EndTime calculates the Duration offset from StartTime. This is robust to wall
+// clock changes between the start and end of the span.
+func (s *SpanData) EndTime() time.Time {
```

Comment:
I don't think we need to provide this. We can simply calculate it from start+dur.